Packs/SplunkPy/Integrations/SplunkPy/SplunkPy_description.md

```diff
@@ -1 +1,3 @@
 Use the SplunkPy integration to fetch incidents from Splunk ES, and query results by SID.
+
+To use Splunk token authentication, set the Username to `_token` and set the Password to `$TOKEN_VALUE`.
```

Comment:
```suggestion
To use Splunk token authentication, enter the text: `_token` to the **Username** field and your token value in the **Password** field.
```
BTW why `_token` and not `token`?

Reply:
as @glicht requested https://github.com/demisto/etc/issues/33995#issuecomment-800964142